backend/slackbot/tests/test.js

```diff
@@ -4,6 +4,114 @@ var assert = require('assert');
 var worker = require('../worker');
```

Comment:
Looks really good overall! I don't see any AWS calls in the functions `extractSlackParameters` and `getSentimentOverTime`, although I could have missed something - is there a reason those functions aren't included? If not, let's add tests for those as well.

Reply:
Yup, I can write tests for those two as well, the only reason I didn't do getSentimentOverTime and report are due to that other than the helper functions (which are tested) there isn't a whole lot of logic, but probably better to test it so I can add those